.github/workflows/main.yml

```diff
@@ -97,32 +106,49 @@ jobs:
           cd ${{env.benchmark_conv_dir}}
           tar -xvzf ${{env.benchmark_tar_dir}}/ptrdist-1.1.tar.gz
           cd ptrdist-1.1
-          for i in anagram bc ft ks yacr2 ; do cd $i ; bear make LOCAL_CFLAGS="-std=c99" ; cd .. ; done
+          for i in anagram bc ft ks yacr2 ; do\
+            (cd $i ; bear make LOCAL_CFLAGS="-std=c99 -Wno-error")\
```

Comment:
Why did you add `-Wno-error`?

Reply:
FTR: [This was to make PtrDist pass](https://github.com/correctcomputation/actions/pull/5/commits/745b7a2ce8393ee0dd21c521ee72078d7286d82f).